mlrun/datastore/targets.py

```diff
@@ -1069,6 +1064,36 @@ def prepare_spark_df(self, df, key_columns, timestamp_key=None, spark_options=No
         return df
 
 
+class ParquetTargetStoreyWrapper(storey.ParquetTarget):
```

Comment:
@alxtkr77 is going to consider wrapping these targets as well: ParquetTarget, CSVTarget, StreamTarget, KafkaTarget, TDEngineTarget.

Reply:
I looked into it - no need